storybook/pages/InlineNetworksComboBoxPage.qml

```diff
@@ -0,0 +1,66 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import Models 1.0
+import utils 1.0
+import AppLayouts.Chat.controls.community 1.0
+
+Item {
+    id: root
+
+    readonly property var modelData: [
```

Comment:
We have `NetworksModel.qml` already created. Could we reuse it here?

Reply:
As in the previous comment :)